s4/clarity/instrument.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Semaphore Solutions, Inc.
+# ---------------------------------------------------------------------------
+
+from ._internal import ClarityElement
+from ._internal.props import subnode_property
+from s4.clarity import types, lazy_property
+
+
+class Instrument(ClarityElement):
+    """
+    Reference: https://d10e8rzir0haj8.cloudfront.net/5.3/rest.version.instruments.html
+    Note: xml root limsid attribute e.g. "55-6" does not have use in API.
```

Comment:
This note just reflects standard behaviour for the whole API so it's confusing to have it here.

Reply:
Noted: will fix in the evening.

Reply:
let me know if this doc string is still more distracting/confusing, as opposed to usefully informing.